### ISSUE TITLE
Accessibillity

### ISSUE THREAD

**Issue by Colin-Farkas-Personal** [Colin-Farkas-Personal/Weather-Plants#4]
### Requirements
- The overview page should be accessible with screen readers and through keyboard
